### ISSUE TITLE
Would it be possible to make Kivy apps on iOS with Pythonista and/or Pyto?

### ISSUE THREAD

**Issue by cclauss** [kivy/kivy-ios#766]
**Is your feature request related to a problem? Please describe.**

I use both [Pythonista](https://omz-software.com/pythonista) and [Pyto](https://pyto.app) to develop Python 3.10 apps on iOS.
Both support some level of user interface but nothing as powerful as Kivy. Is it possible to make Kivy apps directly on iOS with one of these tools or do Apple's security guidelines prevent that?

**Describe the solution you'd like**
Fire up `Pythonista` or `Pyto` and build Kivy apps using only my iPad.

**Describe alternatives you've considered**
Going snowboarding.

**Additional context**
Pythonista:
* https://omz-software.com/pythonista/docs
* https://omz-software.com/pythonista/docs/ios/ui.html
* https://forum.omz-software.com
* https://github.com/omz/Pythonista-Issues
* https://testflight.apple.com/join/qDkBu2ur Python 3.10-based beta

Pyto:
* https://github.com/ColdGrub1384/Pyto
* https://discord.gg/XCKkvJS

**Comment by cclauss** [comment]
@misl6 Your thoughts?

**Comment by misl6** [comment]
Going snowboarding looks like a nicer alternative 😄 

Unfortunately, I never used `Pythonista` or `Pyto`, nor do I know how it works under the hood.

Our bootstrap script is quite easy, and also the patching needed decreased over the years, but Kivy uses SDL as a window provider on iOS, which will need to control the window setup.

So yeah, maybe is feasible, even if not that easy, but it will need a lot of cooperation between Pythonista/Pyto and our team.

Would something like https://github.com/kivy/kivy-launcher (works on Android, lacks iOS support but it's not hard to add) will work for your use case?

Basically, the Kivy launcher is a pre-packaged app with all the needed requirements, which is able to run a `.py` script (that you can edit anywhere else) with your App code.

**Comment by cclauss** [comment]
@omz and/or @ColdGrub1384 Any interest in working to build Kivy apps ___directly on iOS___?

**Comment by cclauss** [comment]
@misl6 I _did_ go snowboarding anyway.  I live on the Swiss side of the Stelvio pass so just a bit North of you.  🏂
